rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -180,6 +179,11 @@ ENV
 echo PAI_CURRENT_CONTAINER_PORT="$(cut -f 1 -d"," <<< $PAI_CONTAINER_HOST_http_PORT_LIST)" >> $ENV_LIST
 
 
+# Prepare docker debug log
+mkdir -p "/tmp/pai-root/log/$APP_ID/$CONTAINER_ID"
+ln -s /tmp/pai-root/log/$APP_ID/$CONTAINER_ID/DockerContainerDebug.log $LOG_DIRS/DockerContainerDebug.log
```

Comment:
@yqwang-ms mounting `/var/lib/yarn/userlogs/...` as a volume in `docker run` does not work, so using symbolic link instead